### ISSUE TITLE
OSX Touchpad Gesture Navigation

### ISSUE THREAD

**Issue by afollestad** [liri-browser/liri-browser#67]
In Chrome, Safari, Firefox, etc. on a Mac, you can swipe from left to right with two fingers on the track pad to go back (and right to left to go forward). 

This is a must for many Mac users.

**Comment by pierremtb** [comment]
That would be an awesome feature to bring to other systems.

**Comment by VirusKA** [comment]
I agree with @pierremtb. If we can get this to Papyros, that'll be amazing.
